lib/matplotlib/tests/test_quiver.py

```diff
@@ -43,9 +44,9 @@ def test_quiver_key_memory_leak():
     qk = ax.quiverkey(Q, 0.5, 0.92, 2, r'$2 \frac{m}{s}$',
                       labelpos='W',
                       fontproperties={'weight': 'bold'})
-    assert sys.getrefcount(qk) == 3
+    orig_refcount = sys.getrefcount(qk)
     qk.remove()
-    assert sys.getrefcount(qk) == 2
+    assert sys.getrefcount(qk) < orig_refcount
```

Comment:
```suggestion
    assert sys.getrefcount(qk) == orig_refcount - 1
```
? If not, feel free to self merge.